runtime/oti/vmconstantpool.xml

```diff
@@ -340,6 +348,30 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 
 	<fieldref class="com/ibm/oti/util/WeakReferenceNode" name="next" signature="Lcom/ibm/oti/util/WeakReferenceNode;" flags="opt_methodHandle"/>
 
+	<!-- OpenJDK MethodHandles: field references. -->
+	<fieldref class="java/lang/invoke/MemberName" name="clazz" signature="Ljava/lang/Class;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MemberName" name="name" signature="Ljava/lang/String;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MemberName" name="type" signature="Ljava/lang/Object;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MemberName" name="flags" signature="I" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MemberName" name="resolution" signature="Ljava/lang/Object;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MethodType" name="ptypes" signature="[Ljava/lang/Class;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MethodType" name="rtype" signature="Ljava/lang/Class;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MethodType" name="form" signature="Ljava/lang/invoke/MethodTypeForm;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MethodTypeForm" name="parameterSlotCount" signature="S" versions="14-" flags="opt_openjdkMethodhandle">
+		<fieldalias name="argCounts" signature="J" versions="8-11"  flags="opt_openjdkMethodhandle"/>
+	</fieldref>
+	<fieldref class="java/lang/invoke/LambdaForm" name="vmentry" signature="Ljava/lang/invoke/MemberName;" flags="opt_openjdkMethodhandle"/>
+	<fieldref class="java/lang/invoke/MethodHandle" name="form" signature="Ljava/lang/invoke/LambdaForm;" flags="opt_openjdkMethodhandle"/>
+		
+	<!--
+	Field references used by both OpenJ9 and OpenJDK MethodHandles are mentioned below.
+	Multiple flags or multiple definitions under different flags are unsupported for field references.
+	So, the below field references are defined without a flag field.
+	These field references are only needed for opt_methodHandle and opt_openjdkMethodhandle flags.
+	-->
+	<fieldref class="java/lang/invoke/MethodType" name="methodDescriptor" signature="Ljava/lang/String;"/>
```

Comment:
Another alternative is to add an `opt_methodHandleCommon` flag that is required by both `opt_methodHandle` and `opt_openjdkMethodhandle`.

This comment can be moved to the description of that flag and it makes it clear (and enforced) the dependency between them

Reply:
This should work. I will add another flag for common dependencies between OpenJ9 and OpenJDK MethodHandles.

Reply:
Added the `opt_methodHandleCommon` flag.